### PR TITLE
Jaycam/benefits section

### PR DESCRIPTION
**Pull request by CamilingJS**
Issue: #<Number>
#12
## Summary (1-2 sentences)
Created the 3 column benefits informative section. They are links to the routes. 
## Details (reason and description of the changes)
This format is a basic landing page feature 

## Screenshots/Recordings (if applicable)
image

## Does it work on mobile? Make sure to include mobile screenshots or recordings!

Yes it is flex-wrapped
image

### CODE REVIEW

File: app/components/marketing/benefits-section.tsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
+const benefits = [
```
Comment (andrelandgraf) [inline app/components/marketing/benefits-section.tsx:1]: Nice approach!

File: app/components/marketing/benefits-section.tsx [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    {
+        title: "Join a group",
+        message: "Do what you love, meet others who love it, find your community. The rest is history!",
+        imgUrl: "https://res.cloudinary.com/dxctpvd8v/image/upload/v1687225039/SocialPlan-it/Feature-Image-1_clboti.png", 
+        imgAlt: "image description",
```
Comment (andrelandgraf) [inline app/components/marketing/benefits-section.tsx:6]: Maybe add a short description alt text for every image!

**Review by andrelandgraf — COMMENTED** [review]
Great work! Left just one comment!

**Review by willbeaumont — DISMISSED** [review]
Commented in the discord wondering if just updating the alt text for these images is ok, or bad manners in a PR.

File: app/components/marketing/benefits-section.tsx [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+        imgUrl: "https://res.cloudinary.com/dxctpvd8v/image/upload/v1687225039/SocialPlan-it/Feature-Image-2_c5mgcb.png", 
+        imgAlt: "image description",
+        link: "/events", 
+
+    },
+    {
+        title: "Start a group",
+        message: "You don’t have to be an expert to gather people together and explore shared interests.",
+        imgUrl: "https://res.cloudinary.com/dxctpvd8v/image/upload/v1687225039/SocialPlan-it/Feature-Image-3_vpibsv.png", 
+        imgAlt: "image description",
```
Comment (andrelandgraf) [inline app/components/marketing/benefits-section.tsx:21]:
```suggestion
        imgAlt: "A group of people smiling into the camera.",
```

File: app/components/marketing/benefits-section.tsx (hunk 2 above)
Comment (CamilingJS) [reply app/components/marketing/benefits-section.tsx]: > Maybe add a short description alt text for every image!

This is done!

File: app/components/marketing/benefits-section.tsx [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+        imgAlt: "Five members sitting on a ledge bench",
+        link: "/events", 
+
+    },
+    {
+        title: "Start a group",
+        message: "You don’t have to be an expert to gather people together and explore shared interests.",
+        imgUrl: "https://res.cloudinary.com/dxctpvd8v/image/upload/v1687225039/SocialPlan-it/Feature-Image-3_vpibsv.png", 
+        imgAlt: "Multiple people conversing on the steps",
+        link: "/groups",
```
[the remark this answers is not in the document]
Comment (CamilingJS) [reply app/components/marketing/benefits-section.tsx]: This is done! Thank you!

**Review by tonydangblog — APPROVED** [review]
Everything looks good to me. Let's get it merged! 🙂

**Review by shami-dev — APPROVED** [review]
Looks good!

**Pull request merged**